<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MiningOverviewFrame</class>
 <widget class="QFrame" name="MiningOverviewFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>24</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>24</number>
   </property>
   <property name="verticalSpacing">
    <number>12</number>
   </property>
   <item row="6" column="3">
    <widget class="QLabel" name="m_overviewHashrateLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="m_miningStatusTextLabel">
     <property name="text">
      <string>Mining status</string>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="1" colspan="4">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>16</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resources.qrc">:/icons/mining</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_titleLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string>MINING DETAILS</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="m_overviewHashrateTextLabel">
     <property name="text">
      <string>Your hashrate</string>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QLabel" name="m_overviewNetworkDifficultyLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QLabel" name="m_overviewNetworkHashrateTextLabel">
     <property name="text">
      <string>Network hashrate</string>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QLabel" name="m_overviewNetworkHashrateLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QLabel" name="m_miningStateLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>OFF</string>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QLabel" name="m_overviewNetworkDifficultyTextLabel">
     <property name="text">
      <string>Difficulty</string>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QLabel" name="m_CopyAffiliateLinkLable">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/icons/copy</pixmap>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="QPushButton" name="m_CopyAffiliateLinkButton">
     <property name="text">
      <string>Copy my affiliate link to clipboard </string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>startMiningClicked()</slot>
 </slots>
</ui>
